include/xtensor-io/xnpz.hpp

```diff
@@ -39,6 +39,98 @@ namespace xt
 {
     using namespace std::string_literals;
 
+    namespace detail
+    {
+#pragma pack(push, 2)
```

Comment:
Why is this required? Besides, this is Windows specific stuff so it should be guarded.